recipes/valgrind/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "valgrind" %}
+{% set version = "3.13.0" %}
+{% set sha256 = "d76680ef03f00cd5e970bbdcd4e57fb1f6df7d2e2c071635ef2be74790190c3b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://sourceware.org/pub/{{ name }}/{{ name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - pkg-config
+    - perl 5.22*
+    - boost 1.65.1
+    - zlib 1.2.8
+    - openmp
```

Comment:
Do you need `llvm`'s `openmp` here?

Reply:
valgrind's configure checks for it, so I assumed there's a benefit to having it?

Reply:
I assume it checks for `gcc`'s openmp and not `llvm`'s one